alpenhorn/auto_import.py

```diff
@@ -175,12 +184,10 @@ def add_acq(name, node, comment=""):
         raise AlreadyExists("Acquisition \"%s\" already exists in DB." %
                             name)
 
-    # Find an acquisition type that can handle this acq
-    acq_type = ac.AcqType.detect(name, node)
-
-    if acq_type is None:
-        log.debug("No handler available to process \"%s\"" % name)
-        return
+    # TODO: should we still check?
```

Comment:
I think we don't need to check any more at this point. Other than as a validation of arguments (in which case we should raise `ValueErrror`).